.gitignore

```diff
@@ -8,6 +8,7 @@
 **/*.egg-info
 **/*.pyc
 **/dist
+not a tty
```

Comment:
?

Reply:
I think the AWS cli or mfa script drops the file . It appeared somehow